schemas/chemotherapy.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "chemotherapy",
+  "description": "The collection of data elements describing the details of a chemotherapy treatment completed by a donor.",
+  "fields": [
+    {
+      "name": "program_id",
+      "valueType": "string",
+      "description": "Unique identifier of the ARGO program.",
+      "restrictions": {
+        "required": true
+      }
+    },
+    {
+      "name": "submitter_donor_id",
+      "valueType": "string",
+      "description": "Unique identifier of the donor, assigned by the data provider.",
+      "restrictions": {
+        "required": true,
+        "regex": "[A-Za-z0-9\\-\\._]{1,64}"
```

Comment:
If this regex is trying to limit the length of the donor id to 64 characters then there is an issue - specifically, if the id is longer than 64 then the regex wills till pass because it found a string with between 1 and 64 characters... i think we need to modify it to be:
`^[A-Za-z0-9\\-\\._]{1,64}$` Here, the `^` indicating start of string and `$` end of string

Reply:
It's the same regex for all ids - this is taken directly from the hl7 id specification, with the slight addition if a underscore for us.  We can change it to this everywhere, or maybe we can wait for your lecture features to be done.